guide/src/format/theme/syntax-highlighting.md

```diff
@@ -1,16 +1,67 @@
 # Syntax Highlighting
 
-For syntax highlighting I use [Highlight.js](https://highlightjs.org) with a
+For syntax highlighting we use [Highlight.js](https://highlightjs.org) with a
```

Comment:
while we are this, let's remove the subject:  (might have to adjust for line width)

```suggestion
[Highlight.js](https://highlightjs.org) with a custom theme is used for syntax highlighting
```

Reply:
I went with:

> mdBook uses [Highlight.js](https://highlightjs.org) with a custom theme for syntax highlighting.

Reply:
that reads much better than my suggestion :D cool